src/app/pages/wallets/my-wallets/wallet-form-modal/wallet-form-modal.component.ts

```diff
@@ -17,15 +17,35 @@ export class WalletFormModalComponent {
         @Inject(MAT_DIALOG_DATA) public data: MyWallet | undefined,
     ) {
         this.form = new FormGroup<IWalletModalFormType>({
-            name: new FormControl(this.data?.name ?? '', { nonNullable: true }),
+            name: new FormControl(this.data?.name ?? '', {
+              validators: [
+                Validators.required,
+                Validators.maxLength(20),
+              ],
+              nonNullable: true,
+            }),
         });
     }
 
+    public showNameError(): boolean {
+      return this.form.get('name')?.invalid && this.form.get('name')?.touched && this.form.get('name')?.errors?.['required'];
```

Comment:
`showNameError` and `showMaxLength` functions have a lot of in common - please think about a way to abstract away and reuse common part :) Also consider changing those into getters instead methods.